### ISSUE TITLE
Load testing for blazor application

### ISSUE THREAD

**Issue by Rashmiahuja84** [ray440/IcuBlazor#12]
IcuBlazor suppotrs loadtesting?

**Comment by ray440** [comment]
Yes, you can do load testing with IcuBlazor.  Just create a test, run it many times and measure results.

There are many kinds of load testing: server load, client load, db load, etc...  What are you looking for?  What kind of api would be useful to you?

**Comment by Rashmiahuja84** [comment]
We want to do GUI based performance testing simulating user action like
login etc.we want to measure user experience under load conditions. I.e.
response times of each step in our web application. is it possible with
IcuBlazor?

Rashmi Ahuja  | Module Lead
[image: HotelHub-logo]
HotelHub LLP
Phone: +91 9886473716
Email: ***@***.*** ***@***.***>
Website: www.hotelhub.com

------------------------------

HotelHub LLP is a service provider working on behalf of Travel Centric
Technology Ltd, a company registered in the United Kingdom.
DISCLAIMER: This email message and all attachments are confidential and may
contain information that is Privileged, Confidential or exempt from
disclosure under applicable law. If you are not the intended recipient, you
are notified that any dissemination, distribution or copying of
this email is strictly prohibited. If you have received this email in
error, please notify us immediately by return email to
***@***.*** and destroy the original message.
Opinions, conclusions and other information in this message that do not
relate to the official business of Travel Centric Technology Ltd or
HotelHub LLP, shall be understood to be neither given nor endorsed by
either company.

**Comment by ray440** [comment]
Yes, you can write automation code to click buttons and login, etc... If you need help you can email me at ray440@gmail.com.

**Comment by Rashmiahuja84** [comment]
Earlier we were using Neoload for AFT testing , Attaching screenshot
for reference what i am looking for.
[image: image.png]

Rashmi Ahuja  | Module Lead
[image: HotelHub-logo]
HotelHub LLP
Phone: +91 9886473716
Email: ***@***.*** ***@***.***>
Website: www.hotelhub.com

------------------------------

HotelHub LLP is a service provider working on behalf of Travel Centric
Technology Ltd, a company registered in the United Kingdom.
DISCLAIMER: This email message and all attachments are confidential and may
contain information that is Privileged, Confidential or exempt from
disclosure under applicable law. If you are not the intended recipient, you
are notified that any dissemination, distribution or copying of
this email is strictly prohibited. If you have received this email in
error, please notify us immediately by return email to
***@***.*** and destroy the original message.
Opinions, conclusions and other information in this message that do not
relate to the official business of Travel Centric Technology Ltd or
HotelHub LLP, shall be understood to be neither given nor endorsed by
either company.

**Comment by ray440** [comment]
The image didn't come thru. But I see.  IcuBlazor is much more low level a tool than neoload. To use it you will need to write code.

**Comment by Rashmiahuja84** [comment]
We are looking to measure user response time of Blazor application during
load. We have written code for automation as functional test which is
working fine but that does not give accurate response time.
[image: image.png]


Rashmi Ahuja  | Module Lead
[image: HotelHub-logo]
HotelHub LLP
Phone: +91 9886473716
Email: ***@***.*** ***@***.***>
Website: www.hotelhub.com

------------------------------

HotelHub LLP is a service provider working on behalf of Travel Centric
Technology Ltd, a company registered in the United Kingdom.
DISCLAIMER: This email message and all attachments are confidential and may
contain information that is Privileged, Confidential or exempt from
disclosure under applicable law. If you are not the intended recipient, you
are notified that any dissemination, distribution or copying of
this email is strictly prohibited. If you have received this email in
error, please notify us immediately by return email to
***@***.*** and destroy the original message.
Opinions, conclusions and other information in this message that do not
relate to the official business of Travel Centric Technology Ltd or
HotelHub LLP, shall be understood to be neither given nor endorsed by
either company.

**Comment by ray440** [comment]
Ok. How are you calculating the response time?

It appears that this GitHub thread is blocking your images. We should move this conversation to email.  What's your email?

**Comment by Rashmiahuja84** [comment]
***@***.***

Rashmi Ahuja  | Module Lead
[image: HotelHub-logo]
HotelHub LLP
Phone: +91 9886473716
Email: ***@***.*** ***@***.***>
Website: www.hotelhub.com

------------------------------

HotelHub LLP is a service provider working on behalf of Travel Centric
Technology Ltd, a company registered in the United Kingdom.
DISCLAIMER: This email message and all attachments are confidential and may
contain information that is Privileged, Confidential or exempt from
disclosure under applicable law. If you are not the intended recipient, you
are notified that any dissemination, distribution or copying of
this email is strictly prohibited. If you have received this email in
error, please notify us immediately by return email to
***@***.*** and destroy the original message.
Opinions, conclusions and other information in this message that do not
relate to the official business of Travel Centric Technology Ltd or
HotelHub LLP, shall be understood to be neither given nor endorsed by
either company.

**Comment by ray440** [comment]
LOL. GitHub is blocking emails too. My email is ray440 at gmail dot com.
